docs/en/engines/table-engines/mergetree-family/versionedcollapsingmergetree.md

```diff
@@ -38,17 +38,10 @@ For a description of query parameters, see the [query description](../../../sql-
 VersionedCollapsingMergeTree(sign, version)
 ```
 
-#### sign
-
-`sign` — Name of the column with the type of row: `1` is a “state” row, `-1` is a “cancel” row.
-
-    The column data type should be `Int8`.
-
-#### version
-
-`version` — Name of the column with the version of the object state.
-
-    The column data type should be `UInt*`.
+| Parameter | Description                                                                            | Type                                                                                                                                                                                                                                                                                    |
+|-----------|----------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `sign`    | Name of the column with the type of row: `1` is a “state” row, `-1` is a “cancel” row. | [`Int8`](/docs/en/sql-reference/data-types/int-uint)                                                                                                                                                                                                                                    |
+| `version` | Name of the column with the version of the object state.                               | [`UInt*`](/docs/en/sql-reference/data-types/int-uint), [`Date`](/docs/en/sql-reference/data-types/date), [`Date32`](/docs/en/sql-reference/data-types/date32), [`DateTime`](/docs/en/sql-reference/data-types/datetime) or [`DateTime64`](/docs/en/sql-reference/data-types/datetime64) |
```

Comment:
The mention to `UInt*` is confusing/misleading to me. Is it a pointer to an UInt? I don't think so... it rather looks like a typo. I see it comes from https://clickhouse.com/docs/en/engines/table-engines/mergetree-family/replacingmergetree#ver. My guess is that the `*` was supposed to be outside as a footnote or something for clarifying something, but maybe it's long shot :bouncing_ball_man: 

We'd rather test it to make sure what needs to be there, modifying the ReplacingMergeTree doc as well if needed.

Reply:
@pamarcos we use `(U)Int*` in quite a few places in the documentation eg. [here](https://clickhouse.com/docs/en/sql-reference/functions/math-functions#sin) to indicate that it can be any of UInt8/16/32 etc or Int8/16/32 etc. Maybe `(U)Int types` would be better?

Reply:
I didn't know that. If we're already using it across our documentation, I'm perfectly fine with it :+1: